pkg/registry/apis/folders/register.go

```diff
@@ -154,42 +154,57 @@ func (b *FolderAPIBuilder) PostProcessOpenAPI(oas *spec3.OpenAPI) (*spec3.OpenAP
 	return oas, nil
 }
 
+type authorizerParams struct {
+	user identity.Requester
+	eval accesscontrol.Evaluator
+	dec  authorizer.Decision
+	req  string
```

Comment:
this is really returning an error message?  if so i would remove this and return an error from the function